src/Appointment/Request.css

```diff
@@ -0,0 +1,5 @@
+.form-group{
```

Comment:
Why are we using this classname? Bootstrap has a similar classname. Should not predefine ```widget```.  Any specific reasons why we are setting this?